src/components/App.tsx

```diff
@@ -44,115 +32,160 @@ export default function App() {
+              <span
+                className={`absolute h-0.5 w-5 bg-white transform transition duration-300 ease-in-out ${
+                  isOpen ? "rotate-45 delay-200" : "-translate-y-1.5"
+                }`}
+              ></span>
+              <span
+                className={`absolute h-0.5 bg-white transform transition-all duration-200 ease-in-out ${
+                  isOpen ? "w-0 opacity-50" : "w-5 delay-200 opacity-100"
+                }`}
+              ></span>
+              <span
+                className={`absolute h-0.5 w-5 bg-white transform transition duration-300 ease-in-out ${
+                  isOpen ? "-rotate-45 delay-200" : "translate-y-1.5"
+                }`}
+              ></span>
+            </div>
+          </button>
+          {capitalizeFirstLetter(chain.name)}
         </div>
-        <div className="md:flex flex-row flex-wrap justify-center lg:pt-8">
-          <Card>
-            <div className="flex justify-between mx-8 py-8 border-b-2 border-gray-200 ">
-              <h1
-                  className="text-2xl lg:text-4xl"
-                  style={{color: extraInfo?.color}}
-              >
-                Metadata #{specs.metadataVersion}
-              </h1>
-              <div className="flex border-2 border-black rounded-xl p-2">
-                {specs.metadataQr.signedBy ? (
+      </div>
+      <div className="flex h-[91vh] md:h-auto">
+        <Sidebar
+          allChains={allChains}
+          sidebarStyle={sidebarStyle}
+          currentNetwork={currentNetwork}
+          setLocalNetwork={setLocalNetwork}
+          setCurrentNetwork={setCurrentNetwork}
+          color={color}
+        />
+        {/** darker layer*/}
+        {isOpen && (
+          <div
+            className="absolute w-full h-[91vh] bg-black opacity-80 z-20 visible"
+            onClick={() => {
+              setIsOpen(!isOpen);
+            }}
+          />
+        )}
+        {/** Main content */}
+        <div className="m-auto flex flex-col md:absolute md:left-60 md:pl-20 md:m-0">
+          <div className="md:flex flex-row flex-wrap justify-center md:pt-8">
+            <div
+              className="px-2 py-2 rounded-lg border-gray-600 bg-white text-black"
+              style={{ minWidth: "25rem" }}
+            >
+              <div className="flex justify-between mx-8 py-8 border-b-2 border-gray-200 items-center">
+                <h1 className="text-lg md:text-2xl" style={{ color }}>
+                  Metadata #{chain.metadataVersion}
+                </h1>
+                <div className="flex border-2 border-black rounded-xl p-2">
+                  {metadataQr.signedBy ? (
                     <div className="text-black font-normal">
-                      <BadgeCheckIcon className={svgClass}/>
-                      Signed by {specs.metadataQr.signedBy}
+                      <BadgeCheckIcon className={svgClass} />
+                      Signed by {metadataQr.signedBy}
```

Comment:
I think it should be green
<img width="598" alt="image" src="https://user-images.githubusercontent.com/7165377/172148533-cc1149f2-b316-4ad6-be20-f871cf42709d.png">
https://www.figma.com/file/eOnz3xbzObTG8hQlIRs6jg/Metadata-Portal?node-id=0%3A1